tools/src/generator.ts

```diff
@@ -0,0 +1,197 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+import { graphql } from '@octokit/graphql';
+
+export const RELEASE_NOTES_SECTION_TAG = '// release-notes';
+
+const DEFAULT_MAPPINGS = new Map<string, string>([
+  ['feat', 'Features'],
+  ['fix', 'Bug Fixes'],
+  ['docs', 'Documentation'],
+  ['chore', 'Other'],
```

Comment:
is chore not already a dedicated section ?
Like 'Chore'. I would say Other is not what is already specified